I/ipx/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ipx"
+version = v"1.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ERGO-Code/ipx.git", "234d484ed1813ce1d95e49c2d86da79d2df18f95"),
+    GitSource("https://github.com/ERGO-Code/basiclu.git", "fe63cd34ab0259329e5a979df7f19ff0de59cfae"),
```

Comment:
Why don't you use it as dependency?

Reply:
I forgot to copy the basiclu header files when I compiled BASICLU....